ssh.go

```diff
@@ -2,6 +2,7 @@ package ssh
 
 import (
 	"crypto/subtle"
+	"golang.org/x/crypto/ssh"
```

Comment:
This should probably be imported as gossh.

Reply:
Nice catch, I've fixed that now